requirements/requirements_dev.txt

```
@@ -8,7 +8,11 @@ pyright==1.1.254
 
 ## linting
 slotscheck~=0.14.0
-
+flake8~=4.0.1
+flake8-bandit~=3.0.0
+flake8-isort~=4.1.1
+flake8-bugbear~=22.4.25
```

```suggestion
flake8-bugbear~=22.7.1
```
update results in new B023 error

---

https://github.com/DisnakeDev/disnake/pull/557/commits/03b7ee641befddae754ac11d35c1f4196a33d35d